nt-fe/components/ui/alert.tsx

```diff
@@ -11,7 +11,9 @@ const alertVariants = cva(
         default: "bg-card text-card-foreground",
         info: "bg-general-info-background-faded text-general-info-foreground border border-general-info-border",
         destructive:
-          "text-destructive bg-card [&>svg]:text-current *:data-[slot=alert-description]:text-destructive/90",
+          "bg-red-50 dark:bg-red-950/50 border-red-200 dark:border-red-800 text-red-800 dark:text-red-200 [&>svg]:text-red-600 dark:[&>svg]:text-red-400",
+        warning:
+          "bg-yellow-50 dark:bg-yellow-950/50 border-yellow-200 dark:border-yellow-800 text-yellow-800 dark:text-yellow-200 [&>svg]:text-yellow-600 dark:[&>svg]:text-yellow-400",
```

Comment:
Our UI has different error and warning state, so I have added these colors, do you think we should stay with library one or our own implementation? @akorchyn

Reply:
Ideally, we should create a wrapper around the original alert with an override similar to @/components/button, overrides components/ui/button.

We don't want to really change the default implementation, as we might, from time to time, update that component, and it will override our settings.